src/common/authorization.rs

```diff
@@ -168,8 +172,10 @@ impl Credentials for Basic {
     }
 
     fn encode(&self) -> HeaderValue {
+        const ENGINE: FastPortable = FastPortable::from(&STANDARD, FastPortableConfig::new());
+
         let mut encoded = String::from("Basic ");
-        base64::encode_config_buf(&self.decoded, base64::STANDARD, &mut encoded);
+        base64::encode_engine_string(&self.decoded, &mut encoded, &ENGINE);
```

Comment:
You should be able to use [`DEFAULT_ENGINE`](https://docs.rs/base64/0.20.0/base64/engine/constant.DEFAULT_ENGINE.html) here